scripts/build-json/build-page-json.js

```diff
@@ -47,21 +39,26 @@ async function buildPageJSON(elementRelativePath) {
     const prosePath = path.join(elementPath, 'prose.md');
     const contributorsPath = path.join(elementPath, 'contributors.md');
 
-    // make the package
-    const element = {};
-    element.title = meta.title;
-    element.mdn_url = meta['mdn-url'];
-    element.interactive_example_url = meta['interactive-example'];
-    element.browser_compatibility = bcd.package(meta['browser-compatibility']);
+    // set up element data
+    const element = {
+      data: {},
+      metadata: {}
+    };
+    element.data.title = meta.title;
```

Comment:
Why is GitHub's javascript syntax highligher lighting up on the `data` key thing but not on `metadata`? It's not a builtin as far as I know.